ow_lander/urdf/lander.xacro

```diff
@@ -548,10 +548,12 @@
         <contact>
           <collide_bitmask>0x0001</collide_bitmask> <!-- Enable collision with the terrain by matching the bitmask -->
           <ode>
-            <kd>1e12</kd>
+            <kd>5e12</kd>
+            <!-- <kp>1e20</kp> -->
           </ode>
         </contact>
       </surface>
+      <max_contacts>1</max_contacts>

```

Comment:
I thought that we discussed that reducing the number of contacts to 1 even though hides the LCP error message but is unlikely good for the accuracy of the results. Do you have new findings on this?

Reply:
As I said, this PR wasn't meant for a code review.. I was just testing if things improved reducing the number of contacts.. I am not planning to use this in the final version of this branch. I'll push the final version soon.